### PR TITLE
docs: add engagement rules

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do? \[optional\]

Specify what projects and how to integrate
- CPU: anyone
- GPU: min 100+ GH stars

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +70,14 @@ Here are pre-requisites for your project before adding to the Lightning EcoSyste
...
 
-(Optional). \[wip\] join our Slack channel to be notified if your project is breaking
+### Additional suggestions and engagement rules
+
+- For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)
+- (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
+- (**Kind request**) include Lightning badge in your readme:
+  ```md
+  [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
+  ```
```
Comment (rohitgr7) [inline README.md:80]:
```suggestion
- For the qualification to use GPU machine we set min number of GitHub stars as 100+ (please note it is for capacity reasons and this threshold may change in the future :)
- (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
- (**Kind request**) Include [Lightning badge](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white) in your readme:
  ```md
  [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
  ```
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
-- [ ] Did you **add your config to CI** - GitHub action and Azure pipeline?
+- [ ] Did you **set `runtimes` in config** for GitHub action integration?
+- [ ] Did you **add your config to CI** in Azure pipeline for 100\* stars?
 - [ ] Are all integration **tests passing**?
 
 ## What does this PR do? \[optional\]
 
-Fixes # (issue).
+Fixes # (issue) \[add link to created issue in your repository if any\]
+OR link your project for clarity.
```
Comment (rohitgr7) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
Fixes # (issue) \[add a link to the created issue in your repository if any\]
OR link your project for clarity.
```

**Review by rohitgr7 — APPROVED** [review]
:nit

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,12 +2,14 @@
 
 - [ ] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
 - [ ] Did you create/update your **configuration file**?
-- [ ] Did you **add your config to CI** - GitHub action and Azure pipeline?
+- [ ] Did you **set `runtimes` in config** for GitHub action integration?
+- [ ] Did you **add your config to CI** in Azure pipeline for 100\* stars?
```
Comment (SkafteNicki) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ github stars)?
```

File: README.md [hunk 4]
```diff
@@ -69,7 +70,14 @@ Here are pre-requisites for your project before adding to the Lightning EcoSyste
    ```
 1. Finally, create a draft PR to the repo!
 
-(Optional). \[wip\] join our Slack channel to be notified if your project is breaking
+### Additional suggestions and engagement rules
+
+- For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)
```
Comment (SkafteNicki) [inline README.md:75]:
```suggestion
- For qualification to use GPU machine we set minimum number of GitHub starts as 100+ (please note its for capacity reasons and this threshold may change in the future :] )
```

**Review by SkafteNicki — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (ethanwharris) [inline README.md:75]:
```suggestion
- To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in future)
```

**Review by ethanwharris — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (Borda) [reply README.md]: used https://github.com/PyTorchLightning/ecosystem-ci/pull/34#discussion_r796040348

File: README.md (hunk 1 above)
Comment (Borda) [reply README.md]: not sure what is the change and seems GH marked it as outdated, could you please check it again...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -2,12 +2,14 @@
 
 - [ ] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
 - [ ] Did you create/update your **configuration file**?
-- [ ] Did you **add your config to CI** - GitHub action and Azure pipeline?
+- [ ] Did you **set `runtimes` in config** for GitHub action integration?
+- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ github stars)?
```
Comment (rohitgr7) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
- [ ] Did you **set `runtimes` in config** for GitHub action integration?
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ GitHub stars)?
```

File: README.md [hunk 6]
```diff
@@ -69,7 +70,14 @@ Here are pre-requisites for your project before adding to the Lightning EcoSyste
    ```
 1. Finally, create a draft PR to the repo!
 
-(Optional). \[wip\] join our Slack channel to be notified if your project is breaking
+### Additional suggestions and engagement rules
+
+- To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in future)
```
Comment (rohitgr7) [inline README.md:75]:
```suggestion
- To qualify for GPU machines we require your project to have 100+ GitHub stars (please note that this is for capacity reasons and may change in the future)
```

**Review by rohitgr7 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rohitgr7) [reply README.md]: some typos, can't remember. Grammarly suggested some... looks like they are resolved now

**Pull request merged**